packages/react-server/src/ReactFlightServer.js

```diff
@@ -1036,15 +1098,58 @@ function callLazyInitInDEV(lazy: LazyComponent<any, any>): any {
   return init(payload);
 }
 
+function callWithDebugContextInDEV<A, T>(
+  task: Task,
+  callback: A => T,
+  arg: A,
+): T {
+  // We don't have a Server Component instance associated with this callback and
+  // the nearest context is likely a Client Component being serialized. We create
+  // a fake owner during this callback so we can get the stack trace from it.
+  // This also gets sent to the client as the owner for the replaying log.
+  const componentDebugInfo: ReactComponentInfo = {
+    env: task.environmentName,
+    owner: task.debugOwner,
```

Comment:
Ideally we'd have the name of the component here too. That's why it shows up as `<...>` in the console task. There's two reasons for this:

1) We don't track the "type". In Fizz we track the parent on ComponentStackNode which has this info. We could potentially eagerly create componentDebugInfo for client components in case they error instead of lazily.
2) We don't have a "name" of ClientReferences atm anyway.